spec/i18n_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'i18n/tasks'
+
+RSpec.describe 'I18n' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.